pkg/grapicmd/di/providers.go

```diff
@@ -0,0 +1,81 @@
+package di
+
+import (
+	"github.com/google/go-cloud/wire"
+	"github.com/izumin5210/gex"
+
+	"github.com/izumin5210/grapi/pkg/clui"
+	"github.com/izumin5210/grapi/pkg/command"
+	"github.com/izumin5210/grapi/pkg/grapicmd"
+	"github.com/izumin5210/grapi/pkg/grapicmd/internal/module"
+	"github.com/izumin5210/grapi/pkg/grapicmd/internal/module/generator"
+	"github.com/izumin5210/grapi/pkg/grapicmd/internal/module/script"
+	"github.com/izumin5210/grapi/pkg/grapicmd/internal/usecase"
+)
+
+func ProvideUI(cfg *grapicmd.Config) clui.UI {
+	return clui.New(cfg.OutWriter, cfg.InReader)
+}
+
+func ProvideCommandExecutor(cfg *grapicmd.Config, ui clui.UI) command.Executor {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideCommandExecutor should have comment or be unexported